magclassic/site_sections/magstock.py

```diff
@@ -1,6 +0,0 @@
-from uber.common import *
```

Comment:
This ought to have been removed earlier, since it's clearly a holdover from MAGStock.